tests/Config.test.ts

```diff
@@ -44,30 +44,15 @@ describe('Test of Config', () => {
     });
 
     it ('Test parsing the settings of a file', () => {
-        let config: Config = new Config(null);
-        config.readFromFile(path.resolve(Utils.getInitCWD(), "tests/config.example.yaml"));
-        assert.strictEqual(config.server.address, "");
-        assert.strictEqual(config.server.port.toString(), "3836");
-        assert.strictEqual(config.server.agora_endpoint.toString(), "http://127.0.0.1:2826");
+        let config: Config = new Config();
+        config.readFromFile(path.resolve(process.cwd(), "docs/config.example.yaml"));
+        assert.strictEqual(config.server.address, "0.0.0.0");
+        assert.strictEqual(config.server.port.toString(), "4242");
+        assert.strictEqual(config.server.agora_endpoint.toString(), "http://8.8.8.8:4567");
```

Comment:
It is strange that this syntax is being tested.

Reply:
Fixed 🙈